### PR TITLE
ATO-1036: Update sequence diagrams to show AIS calls

### PR DESCRIPTION
**Pull request by rpjgds**
## What

Sequence diagrams updated to show where AIS is called in the auth-only and identity journeys.

## How to review

Has the happy-path AIS call been documented correctly?
Do you agree that AIS calls should be represented in these happy-path diagrams?

## Checklist

N/A

### CODE REVIEW

File: docs/diagrams/orchestration/auth-only/auth-only.mmd [hunk 1]
```diff
@@ -12,11 +13,12 @@ sequenceDiagram
     auth ->> orch_api : Token response containing access token
     orch_api ->> auth : Userinfo request containing access token to /userinfo (UserInfoHandler.java)
     auth ->> orch_api : Userinfo response
+    orch_api <<->> ais : Check for interventions
```
Comment (billrdunn) [inline docs/diagrams/orchestration/auth-only/auth-only.mmd:16]: This symbol `<<->>` doesn't render properly for me locally

File: docs/diagrams/orchestration/identity/identity.mmd [hunk 2]
```diff
@@ -5,6 +5,7 @@ sequenceDiagram
     participant auth as Authentication
     participant ipv as IPV Core
     participant spot as SPOT
+    participant ais as AIS
```
Comment (billrdunn) [inline docs/diagrams/orchestration/identity/identity.mmd:8]: There's one futher call to AIS needed in this diagram - after the response from IPV

File: docs/diagrams/orchestration/identity/identity.mmd [hunk 3]
```diff
@@ -17,6 +18,7 @@ sequenceDiagram
     auth ->> orch_api : Token response containing access token
     orch_api ->> auth : UserInfo request containing access token to /userinfo
     auth ->> orch_api : UserInfo response
+    orch_api <<->> ais : Check for interventions
```
Comment (billrdunn) [inline docs/diagrams/orchestration/identity/identity.mmd:21]: Same comment as above

File: docs/diagrams/orchestration/identity/identity.mmd [hunk 4]
```diff
@@ -33,6 +35,7 @@ sequenceDiagram
     spot -) orch_api : Return a signed Core Identity JWT
     orch_api ->> orch_api : Store signed Core Identity JWT
     auth ->> orch_api : Poll for updates
+    orch_api <<->> ais : Check for interventions
```
Comment (billrdunn) [inline docs/diagrams/orchestration/identity/identity.mmd:38]: Same as above

File: docs/diagrams/orchestration/auth-only/auth-only.mmd (hunk 1 above)
Comment (rpjgds) [reply docs/diagrams/orchestration/auth-only/auth-only.mmd]: I made them all a dotted single ended line, the double-ender doesn't render in markdown <11.

File: docs/diagrams/orchestration/identity/identity.mmd (hunk 2 above)
Comment (billrdunn) [reply docs/diagrams/orchestration/identity/identity.mmd:8]: Ah yes you're right - we only call getAccountIntervention in IPVCallback if there's an error and we're not going to SPOT, so non happy-path

File: docs/diagrams/orchestration/identity/identity.mmd [hunk 5]
```diff
@@ -33,6 +35,7 @@ sequenceDiagram
     spot -) orch_api : Return a signed Core Identity JWT
     orch_api ->> orch_api : Store signed Core Identity JWT
     auth ->> orch_api : Poll for updates
+    orch_api -->> ais : Check for interventions
```
Comment (billrdunn) [inline docs/diagrams/orchestration/identity/identity.mmd:38]: Should this come after the COMPLETED response?

Comment (rpjgds) [reply docs/diagrams/orchestration/identity/identity.mmd:38]: It looks like Orch check for interventions before returning the COMPLETED response back to Auth.

The response is here https://github.com/govuk-one-login/authentication-api/blob/7f26ad1d332654ec038ad26ba51591a3c868c6fa/ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/ProcessingIdentityHandler.java#L194 - AIS is checked on L181 above

**Review by billrdunn — APPROVED** [review]

**Pull request merged**